.coderabbitai.yaml

```
@@ -0,0 +1,22 @@
+    enabled: true
+    drafts: false
+chat:
+  auto_reply: true
+rules:
+  untranslated_strings:
+    description: "Detect untranslated strings in the code."
+    match:
+      - pattern: '["\'].*["\']' # Matches hardcoded strings
+        exclude:
+          - path: 'res/values/strings.xml' # Exclude the strings.xml file
+    severity: warning
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)